src/components/schemas/statements.yaml

```diff
@@ -0,0 +1,151 @@
+          example: foreign currency
+          nullable: true
+        bank_name:
+          type: string
+          description: |
+            Specifies the bank name of the statement
+          example: American Express
+        is_debit_positive:
+          type: boolean
+          description: |
+            Specifies the representation of the the amount in the statement, if true then the amount represented by |
+            negative sign are credits, and if false then the amount represented by negative sign are debits.
+          example: true
+        currency:
+          type: string
+          description: |
+            Specifies the currency used in the statement.
+          example: USD
+        name:
+          type: string
+          description: |
+            Specifies the mapping name.
+          example: USD
+          nullable: true
+    ignored_line_items:
+      type: array
+      items: 
+        type: string
+      description: >
+        This specifies list of line items to be ignored in the statement.
+      example:
+        - stlisfdfadssd
+  required:
+    - id
+    - statement_mapping
+    - ignored_line_items
+
+preview_out:
+  properties:
+    transactions:
+      type: array
+      items: 
+        type: object
+        properties:
+          description:
+            type: string
+            description: |
+              Specifies the header name of the description column in the statement.
+            nullable: true
+            example: description
+          bank_name:
+            type: string
+            description: |
+              Specifies the bank name of the statement
+            example: American Express
+          vendor:
+            type: string
+            description: |
+              Specifies the vendor of the statement
+            example: Amazon
+          transaction_date:
+            $ref: './fields.yaml#/timestamptz_utc'
+            description: |
+              Specifies the transaction date on the statement.
+          amount:
+            type: string
+            description: |
+              Specifies the amount on the statement.
+            example: 30
+          card_number:
+            type: string
+            description: |
+              Specifies the card number on the statement.
+            example: xxxx-3343
+          unique_id:
+            type: string
+            description: |
+              Specifies the unique code / transaction id on the statement.
+            example: ed0b2aa913c54b3f903a5030208e1872
+          foreign_amount:
+            type: string
+            description: |
+              Specifies the foreign amount on the statement.
+            nullable: true
+            example: 48
+          foreign_currency:
+            type: string
+            description: |
+              Specifies the foreign currency on the statement.
+            nullable: true
+            example: USD
+          transaction_id:
```

Comment:
`transaction_type` is missing @Kulsekar4

Reply:
Added sir